components/brave_rewards/resources/extension/brave_rewards/request_ads_enabled_panel.tsx

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { render } from 'react-dom'
+import { Provider } from 'react-redux'
+import { Store } from 'webext-redux'
+
+import Theme from 'brave-ui/theme/brave-default'
+import { ThemeProvider } from 'styled-components'
+require('emptykit.css')
+require('../../../../../ui/webui/resources/fonts/muli.css')
+require('../../../../../ui/webui/resources/fonts/poppins.css')
+
+import { LocaleContext } from '../../shared/lib/locale_context'
+import { WithThemeVariables } from '../../shared/components/with_theme_variables'
+
+// Components
+import AdsEnablePanel from './components/ads_enable'
+
+// Utils
+import { getMessage } from './background/api/locale_api'
+
+const store: Store<RewardsExtension.State> = new Store({
```

Comment:
Apologies for not thinking of this before, but we should probably not be connecting to the existing rewards panel Redux store, since we don't need it for this UI and it will be going away soon in any case. I'll sketch out a small refactor to remove this dependency.

Reply:
Here's one way to remove the `RewardsExtension.State` dependency by using a pattern similar to the tip dialog and the new "adaptive captcha" panel.

https://github.com/brave/brave-core/commit/f923349aa7c3a9399936c2cb993b2743f9dcc466

Reply:
Thank you, the suggested commit is awesome! I've applied it with a little change:
`rewardsEnabled: !showOnboarding`